src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/CourseActivity.kt

```diff
@@ -23,41 +20,29 @@ import java.time.LocalDate
 annotation class CourseActivityDslMarker
 
 @NomisDataDslMarker
-interface CourseActivityDsl {
-  @CourseActivityPayRateDslMarker
-  fun payRate(
-    iepLevelCode: String = "STD",
-    payBandCode: String = "5",
+interface CourseActivityDsl : CourseScheduleDslApi, CourseActivityPayRateDslApi, CourseScheduleRuleDslApi
```

Comment:
We then inherit from the abstract methods for all child entities so you still have visibility of who the children are.